backend/src/models/User.ts

```diff
@@ -0,0 +1,52 @@
+import { DataTypes } from "sequelize";
+import { sequelize } from "../config/db";
+
+const User = sequelize.define("user", {
+  id: {
+    type: DataTypes.INTEGER.UNSIGNED,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  userName: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  password: {
+    type: DataTypes.STRING,
+    allowNull: false,
+  },
+  firstName: {
+    type: DataTypes.STRING,
+    allowNull: true,
+  },
+  lastName: {
+    type: DataTypes.STRING,
+    allowNull: true,
+  },
+  phoneNumber: {
+    type: DataTypes.STRING,
+    allowNull: true,
+  },
+  birthDate: {
+    type: DataTypes.DATE,
+    allowNull: true,
+  },
+  wallet: {
+    type: DataTypes.INTEGER,
+    allowNull: true,
+  },
+  refreshToken: {
+    type: DataTypes.STRING,
+    allowNull: true,
+  },
```

Comment:
refresh token is stateless just like `access token`. it's better not to persist refresh token for more than one device session expiration and renewal.